submodules/wizard/wizard.js

```diff
@@ -1687,9 +1702,9 @@ define(function(require) {
 			var self = this;
 
 			return _.transform(wizardData.usageAndCallRestrictions.trunkLimits, function(object, value, trunkType) {
-				_.set(object, trunkType + '_trunks', value);
+				_.set(object, trunkType + '_trunks', _.toNumber(value));
 			}, {
-				allow_prepay: true
+				allow_prepay: wizardData.usageAndCallRestrictions.allowPerMinuteCalls
```

Comment:
Are we 100% certain this prop will be set to a boolean value?

Reply:
Yes, unless the usage and call restrictions step is removed. But considering that possibility in the future, I will set a default value for it.

Reply:
Okay, just wanted to be sure.